backend-shared/config/db/gitopsenginecluster_test.go

```diff
@@ -1,162 +1,57 @@
-package db
+package db_test
 
 import (
 	"context"
-	"strings"
-	"testing"
 
-	"github.com/stretchr/testify/assert"
-)
-
-func TestGetGitopsEngineClusterById(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	clusterCredentials := ClusterCredentials{
-		Clustercredentials_cred_id:  "test-cluster-creds-test-1",
-		Host:                        "host",
-		Kube_config:                 "kube-config",
-		Kube_config_context:         "kube-config-context",
-		Serviceaccount_bearer_token: "serviceaccount_bearer_token",
-		Serviceaccount_ns:           "Serviceaccount_ns",
-	}
-
-	gitopsEngineClusterput := GitopsEngineCluster{
-		Gitopsenginecluster_id: "test-fake-cluster-1",
-		Clustercredentials_id:  clusterCredentials.Clustercredentials_cred_id,
-	}
-
-	err = dbq.CreateClusterCredentials(ctx, &clusterCredentials)
-	assert.NoError(t, err)
-
-	err = dbq.CreateGitopsEngineCluster(ctx, &gitopsEngineClusterput)
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	gitopsEngineClusterget := GitopsEngineCluster{
-		Gitopsenginecluster_id: gitopsEngineClusterput.Gitopsenginecluster_id,
-	}
-
-	err = dbq.GetGitopsEngineClusterById(ctx, &gitopsEngineClusterget)
-	if !assert.NoError(t, err) {
-		return
-	}
-	assert.Equal(t, gitopsEngineClusterput, gitopsEngineClusterget)
-
-	//check for inexistent primary key
-
-	gitopsEngineClusterNotExist := GitopsEngineCluster{
```

Comment:
This 'check for inexistent primary key' test looks like it was removed; I recommend moving it over to the Ginkgo version of the test:  while it's very similar to an existing test, it's slightly different: this test verifies that retrieving a GitOpsEngine cluster that couldn't possibly exist fails. (In contrast, the existing test might exist, since it was previously created earlier in the test, and relies on the delete to be successful)